It is assumed that the mean systolic blood pressure is μ = 120 mm Hg. In the Honolulu Heart Study, a sample of n = 100 people had an average systolic blood pressure of 130.1 mm Hg with a standard deviation of 21.21 mm Hg. Is the group significantly different (with respect to systolic blood pressure!) from the regular population?

1. Set up the hypothesis test.
2. Write down all the steps followed for setting up the test.
3. Calculate the test statistic by hand and also code it in Python. It should be 4.76190. What decision can you make based on this calculated value?

In [8]:
import numpy as np
import pandas as pd
import scipy.stats as stats
import matplotlib.pyplot as plt
import seaborn as sns

%matplotlib inline

We have to use a $t$ distribution since we do not have the standard deviation for the whole population. We only have the standard deviation for our sample of $100$ people. For $t$ distribution, the $t$ score is calculated as below:
$$
t = \pm \frac{130.1 - 120}{\frac{21.21}{\sqrt{100}}} \sigma 
$$

In [14]:
t = abs((130.1 - 120)/(21.21/np.sqrt(100))) # just the absolute for simplicity
t

4.761904761904759

Let us assume that the confidence level (CL) $1 - \alpha$ is $95 \%$, i.e., the significance level $\alpha = 5 \%$.
With CL $= 95 \%$, the critical point (the $x$ axis on the normal distribution) is given by the code below: 

In [15]:
stats.t.ppf(0.95, df=99) # the same result can be observed by checking the 99th row and the probability column with value 0.05
# in the table given in the hint table 

1.6603911559963895

Decision: Since $4.76 > 1.66$, it means that the sample lies in significance level ($\alpha$), hence we reject the null hypothesis. In other words, the sample lies outside the confidence interval and hence the rejection.  